serpent-moltres/isotope-distribution/extract.py

```diff
@@ -0,0 +1,125 @@
+import os
+import numpy as np
+import argparse
+import subprocess
+import serpentTools as sT
+
+
+def makePropertiesDir(outdir, filebase, mapFile, unimapFile, serp1=False,
+                      fromMain=False):
+    '''
+    Takes in a mapping from branch names to material temperatures,
+    then makes a properties directory.
+    Serp1 means that the group transfer matrix is transposed.
+
+    '''
+
+    # the required constants for the analysis:
+    goodStuff = ['Chit', 'Diffcoef', 'Nsf', 'Remxs']
```

Comment:
Isn't this script already in moltres? https://github.com/arfc/moltres/blob/devel/python/extractSerpent2GCs.py . You probably should just use the script from moltres rather than copy pasting it into your repo?

Reply:
Yeah, you are right.